adr/ADR-20.md

```diff
@@ -98,6 +100,7 @@ type ObjectStoreConfig struct {
 	Storage     StorageType		// stream storate_type
 	Replicas    int			// stream replicas
 	Placement   Placement		// stream placement
+    Compression bool        // stream compression
```

Comment:
spacing here

Reply:
yeah, fixed. My editor defaulted to spaces here.